packages/react-docsite-components/src/components/PageHeader/PageHeader.tsx

```diff
@@ -39,15 +53,51 @@ const getStyles: IStyleFunction<IPageHeaderStyleProps, IPageHeaderStyles> = prop
 const getClassNames = classNamesFunction<IPageHeaderStyleProps, IPageHeaderStyles>();
 
 const PageHeaderBase: React.FunctionComponent<IPageHeaderProps> = props => {
-  const { pageTitle = 'Page title', pageSubTitle, theme } = props;
+  const {
+    currentVersionData,
+    onVersionMenuClick,
+    pageTitle = 'Page title',
+    pageSubTitle,
+    theme,
+    versionOptions,
+  } = props;
   const styles = getClassNames(getStyles, { theme });
 
+  const monoFont =
```

Comment:
What was the thinking behind having this in a mono font? I'm not so sure about that, though if we keep it you should probably reuse the `codeFontFamily` variable from `CodeSnippet.styles.ts` to keep things consistent.

Reply:
Well, I was just using the same styling that we have in the front page, happy to remove this if needed.

Reply:
Removed.